server/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const app = express();
+const mongoose = require("mongoose");
+const keys = require("./config/keys");
+const cookieSession = require("cookie-session");
+const passport = require("passport");
+const PORT = process.env.PORT || 5000;
+require("./models/User");
+require("./services/passport");
+
+mongoose.connect(keys.mongoURI, () => {
+  console.log("database connected");
+});
+
+app.use(
+  cookieSession({
+    keys: [keys.cookieKey]
+  })
+);
+
+app.use(passport.initialize());
+app.use(passport.session());
+
+require("./routes/authRoute")(app);
+
+if (process.env.NODE_ENV === "production") {
+  app.use(express.static("client/build"));
+  const path = require("path");
+  app.get("*", (req, res) => {
```

Comment:
Why are we sending static files from the server. Shouldn't they be rendered from React itself?

Reply:
Or nginx